downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -328,17 +337,38 @@ Scope to search for groups in an LDAP tree using the django framework for LDAP a
 Used for installing {HubName} with LDAP.
 
 Default = `SUBTREE`
+| *`automationhub_ldap_group_type`* | 
+
+Describes the type of group returned by *automationhub_ldap_group_search*.
+
+This is set dynamically based on the the values of *automationhub_ldap_group_type_params* and *automationhub_ldap_group_type_class*, otherwise it is the default value coming from django-ldap which is 'None'
```

Comment:
```suggestion
This is set dynamically based on the the values of *automationhub_ldap_group_type_params* and *automationhub_ldap_group_type_class*, otherwise it is the default value coming from django-ldap which is 'None'.
```